spec/tree_spec.rb

```diff
@@ -1,14 +1,77 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+describe 'AppleTree' do
+
+  tree = AppleTree.new
```

Comment:
It is best practice to use either a let(:tree) ... here or declare the tree a ``subject``